src/features/Auth/LogInButton.tsx

```diff
@@ -1,14 +1,24 @@
 import React from "react";
 import { Button } from "react-bootstrap";
 import { useTranslation } from "react-i18next";
-import { useAuth0 } from "@auth0/auth0-react";
+
+import * as api from "./Auth.api";
+
+const { REACT_APP_AUTH_REDIRECT_URL } = process.env;
 
 export const LogInButton = () => {
   const { t } = useTranslation();
-  const { loginWithRedirect } = useAuth0();
+
+  const logIn = () => {
+    api.createRequestToken().then((request_token) => {
+      window.location.replace(
+        `https://www.themoviedb.org/authenticate/${request_token}?redirect_to=${REACT_APP_AUTH_REDIRECT_URL}`
```

Comment:
`https://www.themoviedb.org/authenticate/` = move somewhere in constant? Rewrite code with async/await?